### DISCUSSION TITLE
Unable to found "Transformation" Tab in 0.63.13 Version

### DISCUSSION THREAD

**Discussion by Mudassarhu** [airbytehq/airbyte#44886] [General]
Hello,
I have successfully moved the data from Postgres to Clickhouse by using Airbyte. But my table data is showing in JSON format. 
I want to see the data in tabular format. But I unable to find any "Transformation" tab in this version. although I have found the "transformation" tab in 0.40.0 version.
What should i do ? 

Thanks

**Comment by nataliekwong** [comment]
Hi there, basic normalization was removed in favor of Typing and Deduping (You can read more about it here in our [Destination V2 Documentation](https://docs.airbyte.com/using-airbyte/core-concepts/typing-deduping#destinations-v2-example). The tables with-in nested objects will no longer be created by default.

You are welcome to upvote the community discussion to add Typing & Deduping for Clickhouse here: https://github.com/airbytehq/airbyte/discussions/35339 to help prioritize it for our upcoming roadmap.

**Reply by Mudassarhu** [reply]
@nataliekwong Thanks for the prompt reply. 
so, I have to add DBT for transformation now ?

**Reply by nataliekwong** [reply]
Yes, you can reference the guidance here: https://github.com/airbytehq/airbyte/discussions/35840#discussioncomment-10213750 from @marcosmarxm

**Reply by Mudassarhu** [reply]
Thanks

**Comment by nataliekwong** [comment]
Thanks! I'll go ahead and close this discussion.
